persistent/test/Database/Persist/THSpec.hs

```diff
@@ -176,6 +178,7 @@ spec = describe "THSpec" $ do
     SharedPrimaryKeySpec.spec
     SharedPrimaryKeyImportedSpec.spec
     ImplicitIdColSpec.spec
+    MaybeFieldDefsSpec.spec
```

Comment:
okay awesome

This is testing that the TH generation does the right thing for the Haskell side